cmd/collector/collector.go

```diff
@@ -202,6 +239,38 @@ func newCollectorCommand() *cobra.Command {
 	return cmd
 }
 
+func getFlowRecords() []string {
+	return flowRecords
+}
+
+func flowRecordHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
you should require the HTTP method to be GET in this handler:
```go
	if r.Method == "GET" {
		// ...
	} else {
		http.Error(w, "Invalid request method", http.StatusMethodNotAllowed)
	}
```

Reply:
Fixed.